src/amo/pages/Guides/index.js

```diff
@@ -88,7 +88,7 @@ export const getSections = ({
           exploreMore: i18n.gettext(
             'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
           ),
-          exploreUrl: '/collections/mozilla/password-managers/',
+          exploreUrl: '/collections/4757633/password-managers/',
```

Comment:
should this be hardcoded or is it better to use `config.get('mozillaUserId')`?

Reply:
The URLs are already hardcoded anyway, so I decided to write a simple patch.

Reply:
good point

Reply:
Note: maybe we could to better in the future but we needed a simple patch to cherry-pick for this week's tag.